ssl/ssl_sess.c

```diff
@@ -139,8 +139,11 @@ SSL_SESSION *ssl_session_dup(SSL_SESSION *src, int ticket)
     dest->references = 1;
 
     dest->lock = CRYPTO_THREAD_lock_new();
-    if (dest->lock == NULL)
-        goto err;
+    if (dest->lock == NULL) {
+	OPENSSL_free(dest);
+	dest = NULL;
+	goto err;
```

Comment:
whitespace error: you've got tabs here instead of spaces.

Reply:
@t8m @paulidale I thought we had a source-code checker which catches such problems?

Reply:
> whitespace error: you've got tabs here instead of spaces.

sorry，My ide has this problem and I have fixed it.

Reply:
> @t8m @paulidale I thought we had a source-code checker which catches such problems? (I forgot its name.)

It is not run in CI because it still produces some false positives.